### ISSUE TITLE
org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unknown keras weight initializer GlorotUniform

### ISSUE THREAD

**Issue by sai28593** [eclipse/deeplearning4j#7556]
#### Issue Description
Getting below error when trying to load model(.h5) 
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unknown keras weight initializer GlorotUniform. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasInitilizationUtils.mapWeightInitialization(KerasInitilizationUtils.java:183)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasInitilizationUtils.getWeightInitFromConfig(KerasInitilizationUtils.java:223)
	at org.deeplearning4j.nn.modelimport.keras.layers.core.KerasDense.<init>(KerasDense.java:100)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:219)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:218)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:111)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:58)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildSequential(KerasModelBuilder.java:322)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:195)
	at Predictor.main(Predictor.java:21)

######Code

String simpleMlp = new ClassPathResource(
            "class.h5").getFile().getPath();
MultiLayerNetwork model = KerasModelImport.
                importKerasSequentialModelAndWeights(simpleMlp);

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version - 1.0.0-beta3
* platform information (OS, etc) - MAC
*keras version -  2.2.4

**Comment by sai28593** [comment]
@maxpumperla 
below is the code snipet
import numpy as np
import cv2
import os
import tensorflow as tf

os.environ['KMP_DUPLICATE_LIB_OK']='True'


train_data_dir = "/Users/Downloads/fruits-test/Training"
test_data_dir = "/Users/Downloads/fruits-test/Test"

categories = ["Orange", "Limes", "Banana"]

IMG_SIZE = 100
def create_training_data(datadir):
        training_data = []
        for category in categories:
            path = os.path.join(datadir, category)
            class_num = categories.index(category)
            for img in os.listdir(path):
                img_array = cv2.imread(os.path.join(path, img), cv2.IMREAD_GRAYSCALE)
                new_array = cv2.resize(img_array, (IMG_SIZE,IMG_SIZE))
                training_data.append([new_array, class_num])
        return training_data
    
trained_data = create_training_data(train_data_dir)
test_data = create_training_data(test_data_dir)

import random
random.shuffle(trained_data)
random.shuffle(test_data)

X_train = []
y_train = []

X_test = []
y_test = []



for features, label in trained_data:
    X_train.append(features)
    y_train.append(label)
for features, label in test_data:
    X_test.append(features)
    y_test.append(label)

    
X_train = np.array(X_train).reshape(-1, IMG_SIZE, IMG_SIZE, 1)
X_test = np.array(X_test).reshape(-1, IMG_SIZE, IMG_SIZE, 1)
X_train =  X_train.squeeze()

X_test =  X_test.squeeze()



X_train = tf.keras.utils.normalize(X_train, axis = 1)
X_test = tf.keras.utils.normalize(X_test, axis = 1)

model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Flatten())
 #first parameter no of neurons
 
model.add(tf.keras.layers.Dense(128, activation=tf.nn.relu))
model.add(tf.keras.layers.Dense(128, activation=tf.nn.relu))
model.add(tf.keras.layers.Dense(10, activation=tf.nn.softmax))

model.compile(optimizer = 'adam',
             loss = 'sparse_categorical_crossentropy',
             metrics = ['accuracy'])
model.fit(x = X_train, y = np.asarray(y_train), epochs=3)

val_loss, val_acc = model.evaluate(X_test, np.asarray(y_test))#
print(val_loss, val_acc)

model.save('/Users/s0c01rq/Downloads/fruit_classifer.h5')

#import tensorflow as tf


#new_model =  tf.keras.models.load_model('/Users/s0c01rq/Downloads/fruit_classifer.model')
#predictions = new_model.predict(X_test)
#
##print(predictions[0])
##print(np.argmax(predictions[0]))
#np.argmax(predictions[0])
#
#print(y_test[0])
##plt.imshow(X_test[0])
##plt.show()
#
#
#img = X_test[0]
#print(img.shape)
#
#print(X_test)
#img = (np.expand_dims(img,0))



#predictions_single = new_model.predict(img)


# =============================================================================
# =============================================================================
# def plot_value_array(i, predictions_array, true_label):
#    predictions_array, true_label = predictions_array[i], true_label[i]
#    plt.grid(False)
#    plt.xticks([])
#    plt.yticks([])
#    thisplot = plt.bar(range(10), predictions_array, color="#777777")
#    plt.ylim([0, 1]) 
#    predicted_label = np.argmax(predictions_array)
#   
#    thisplot[predicted_label].set_color('red')
#    thisplot[true_label].set_color('blue')
# 
# 
# plot_value_array(0, predictions_single, y_test)
# _ = plt.xticks(range(10), categories, rotation=45) 
# =============================================================================
# =============================================================================

**Comment by agibsonccc** [comment]
Likely out of date by now. For those in the future, please open a new issue if this still comes up.
